sql/core/src/test/scala/org/apache/spark/sql/errors/QueryExecutionAnsiErrorsSuite.scala

```diff
@@ -403,4 +403,16 @@ class QueryExecutionAnsiErrorsSuite extends QueryTest
       parameters = Map("timeZone" -> "-100")
     )
   }
+
+  test("SPARK-50290: Add a flag to disable DataFrame context") {
```

Comment:
Could you move this test to the more specific suite: `QueryContextSuite`, please.

Reply:
Done.